src/cunumeric/index/advanced_indexing.cu

```diff
@@ -94,7 +94,7 @@ struct AdvancedIndexingImplBody<VariantKind::GPU, CODE, DIM, OUT_TYPE> {
 
     const size_t blocks = (volume + THREADS_PER_BLOCK - 1) / THREADS_PER_BLOCK;
 
-    size_t shmem_size = THREADS_PER_BLOCK / 32 * sizeof(int64_t);
+    size_t shmem_size = THREADS_PER_BLOCK / 32 * sizeof(bool);
```

Comment:
Here the reduction accumulator is of type `uint64_t`, so we need to allocate 8 bytes per warp. The type used in the `sizeof` was technically incorrect (`int64_t`, it should be `uint64_t`), but I already handled this in https://github.com/nv-legate/cunumeric/pull/659, so no changes are needed to this file.

Reply:
Thank you, Manolis!